test/python/test_local_simulator.py

```diff
@@ -1,57 +1,54 @@
-#!/usr/bin/env python
-import unittest
-import time
-import os
-import sys
-import io
-import logging
+# -*- coding: utf-8 -*-
+
+# Copyright 2017 IBM RESEARCH. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# =============================================================================
+
 import random
-import qiskit
-from qiskit import QuantumProgram
-import qiskit.qasm as qasm
-import qiskit.unroll as unroll
+import unittest
+
+from qiskit import qasm, unroll, QuantumProgram
 
 from qiskit.simulators import _localsimulator
 
-class LocalSimulatorTest(unittest.TestCase):
+from .common import QiskitTestCase
+
+
+class LocalSimulatorTest(QiskitTestCase):
     """
     Test interface to local simulators.
     """
-
-    @classmethod
-    def setUpClass(cls):
-        cls.moduleName = os.path.splitext(__file__)[0]
-        cls.log = logging.getLogger(__name__)
-        cls.log.setLevel(logging.INFO)
-        logFileName = cls.moduleName + '.log'
-        handler = logging.FileHandler(logFileName)
-        handler.setLevel(logging.INFO)
-        log_fmt = ('{}.%(funcName)s:%(levelname)s:%(asctime)s:'
-                   ' %(message)s'.format(cls.__name__))
-        formatter = logging.Formatter(log_fmt)
-        handler.setFormatter(formatter)
-        cls.log.addHandler(handler)
-
     @classmethod
     def tearDownClass(cls):
-        #cls.pdf.close()
+        # cls.pdf.close()
```

Comment:
What about removing this line?

Reply:
Sure! I did not cleanup those kind of comments in case some of them served as TODOs or other convention, but I'll remove that one and we can do another pass later for the rest if needed!